code/messaging/src/main/java/com/adobe/marketing/mobile/aepcomposeui/style/AepButtonStyle.kt

```diff
@@ -37,6 +37,7 @@ class AepButtonStyle(
     var border: BorderStroke? = null,
     var colors: ButtonColors? = null,
     var contentPadding: PaddingValues? = null,
+    var buttonTextStyle: AepTextStyle? = null
```

Comment:
nit: don't forget to add this param to the kdoc. i'll get it updated in the repo docs as well.